### ISSUE TITLE
Sample Plot Lidar

### ISSUE THREAD

**Issue by rafaelmoraes33** [legion1581/go2_webrtc_connect#11]
Hi Legion!

I am communicating and reading the Lidar return, but it comes as an array.
I would like to know if you have any examples or someone who can show how to get this data and plot it and save it in an image.
Thanks!

**Comment by legion1581** [comment]
Yes, check the examples/data_channel/lidar. I have added plot example recently
